Makefile.toml

```diff
@@ -338,7 +407,14 @@ cp . ../../../py39
 cd ../../../
 exec --fail-on-error ${PYTHON} ./get-pip.py
 exec --fail-on-error ${PYTHON} -m pip install wheel flit . ".[test]" ".[ssh-tunnel]"
-cm_run_task generate-resources
+
+# If pyside-setup exists, then assume user wants to build and install pyside6
+if is_path_exists "${WORKSPACE}/pyside-setup/setup.py"
+  cm_run_task get-qt-version
+  cm_run_task build-install-pyside
+else
+  exec --fail-on-error ${PYTHON} -m pip install PySide6==6.3.2
```

Comment:
Hardcoding the version here will diverge from our pyproject.toml

Reply:
Maybe there is a way to conditionalize the installation of this in flit? so it doesn't get installed if we're doing a debug build, but does otherwise? Not sure how that could be conditionalized.

Reply:
Yeah, we could work something out. Maybe the @swift-nav/devinfra team can help take it over. We would certainly want it to be in lockstep with CI first and foremost.

Reply:
I think a global variable is a good first step, so we don't have to update the version in multiple places.